bigbluebutton-html5/imports/startup/client/intl.jsx

```diff
@@ -111,19 +113,29 @@ class IntlStartup extends Component {
         .then(({ messages, normalizedLocale }) => {
           const dasherizedLocale = normalizedLocale.replace('_', '-');
           this.setState({ messages, fetching: false, normalizedLocale: dasherizedLocale }, () => {
-            Settings.application.locale = dasherizedLocale;
-            Settings.save();
+            this.saveLocale(dasherizedLocale);
           });
         })
         .catch(() => {
           this.setState({ fetching: false, normalizedLocale: null }, () => {
-            Settings.application.locale = DEFAULT_LANGUAGE;
-            Settings.save();
+            this.saveLocale(DEFAULT_LANGUAGE);
           });
         });
     });
   }
 
+  saveLocale(localeName) {
+    Settings.application.locale = localeName;
+    if (RTL_LANGUAGES.includes(localeName)) {
```

Comment:
Is anything extra needed to support a future `ar_TN` locale? I see that `const RTL_LANGUAGES = ['ar', 'he', 'fa'];` are only the first 2 letters, so `localeName` might be `ar_TN` while `RTL_LANGUAGES` is just `'ar'`